packages/backend/src/open_payments/payment_pointer/model.test.ts

```diff
@@ -231,7 +235,7 @@ export const getRouteTests = <M extends PaymentPointerSubresource>({
     createModel,
     testGet: async ({ id, paymentPointerId, clientId }, expectedMatch) => {
       const paymentPointer = await getPaymentPointer()
-      paymentPointer.id = paymentPointerId
+      paymentPointer.id = paymentPointerId as string
```

Comment:
I had to take a long look at this one.

Could you instead change
https://github.com/interledger/rafiki/blob/421d89ac7883cec8d0f9f5a0d31cda98b6a9a448/packages/backend/src/open_payments/payment_pointer/model.test.ts#L42-L44
to 
```
interface TestGetOptions extends GetOptions {
  paymentPointerId: NonNullable<GetOptions['paymentPointerId']>
}

interface BaseTestsOptions<M> {
  createModel: (options: { clientId?: string }) => Promise<M>
  testGet: (options: TestGetOptions, expectedMatch?: M) => void
```